cmd/container-manager/manage_ctr_mgr.sh

```diff
@@ -113,7 +113,7 @@ install_docker(){
 			sudo -E dnf config-manager --add-repo "$repo_url"
 			sudo -E dnf makecache
 			sudo -E dnf -y install "${pkg_name}"
-		elif [ "$ID" == "centos" ] || [ "$ID" == "rhel" ]; then
+		elif [ "$ID" == "centos" ] || [ "$ID" == "alinux" ] || [ "$ID" == "rhel" ]; then
 			sudo -E yum install -y yum-utils
 			repo_url="https://download.docker.com/linux/centos/docker-ce.repo"
```

Comment:
The repos versioning is identical to rhe/centosl? this repo works?

Reply:
@snir911 Yes, alinux is based CentOS